metricbeat/module/kubernetes/state_node/_meta/test/kube-state-metrics.expected

```diff
@@ -1,36 +1,36 @@
 [
 	{
-		"RootFields": null,
+		"RootFields": {},
```

Comment:
This is the only changed related to this PR, this is caused by `TransformMapStrToEvent`.

Rest of changes also happen on master if file is regenerated.